src/graphql/account.rs

```diff
@@ -287,78 +287,31 @@ impl AccountMutation {
         Ok(username)
     }
 
-    /// Authenticates with the given username and password
+    /// - Authenticates with the given username and password.
+    /// - If the `last_signin_time` value of the `account` is `None`, the operation will fail, and
+    /// it should be guided to call common_sign_in_logic.
```

Comment:
Since this is a open method to users, we need to fulfill the Rust documentation according to [the convention](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html)